google/api/http.proto

```diff
@@ -346,6 +339,18 @@ message HttpRule {
   // message type.
   string body = 7;
 
+  // The name of fields whose values are mapped to HTTP headers in a request. If
```

Comment:
```suggestion
  // A mapping from HTTP headers to request fields. If
```

The text this would replace is describing the keys of the map, rather than the map itself.  It's also a little redundant with the second sentence.